### PR TITLE
Responses.py cleanup

### PR DESCRIPTION
**Pull request by HubbeKing**
It's TIME to clean this bad boi up some.

- [x] get Boops out into its own module
- [x] get Animals out into its own module
- [x] clean up what remains into a WeirdResponses module or something
- [x] store responses in bot storage?
- [x] tests?

Help appreciated

### CODE REVIEW

**Comment by HubbeKing** [comment]
H'okay, this is about as good as I can make it right now. Might look at the Weirder responses later, but feel free to finish up if you like.

**Review by Heufneutje — APPROVED** [review]

**Comment by HubbeKing** [comment]
Okay, I did a basic refactor on the remaining stuff in Responses.py
(I also threw out the Ponies response, as the link it had was a 404, and the DuctTape response, as that hasn't been relevant for YEARS. Feel free to prune further in datastore_default.json)

This is now about done, I think? Unless either of you have Good Ideas for how to make this less gross?

File: desertbot/datastore_default.json [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+      "14": "{user} might be the droid you are looking for.",
+      "19": "{user} is the droid you are looking for.",
+      "20": "{user} is DEFINITELY the droid you are looking for."
+    },
+    "goose": {
+      "1": "{user} is a clown!",
+      "8": "{user} is not {article} {animal}.",
+      "14": "{user} /might/ be {article} {animal}.",
+      "19": "{user} is DEFINITELY {article} {animal}.",
+      "20": "{user} is {article} CRITICAL {animal}!"
```
Comment (StarlitGhost) [inline desertbot/datastore_default.json]: "it's a lovely day in the chat, and {user} is a horrible goose." for critical

File: desertbot/modules/automatic/Animals.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    def help(self, arg):
+        return 'Responds to animal noises.'
+
+    def onLoad(self) -> None:
+        defaultReactions = {
+            "1": "{user} critically fails at being {article} {animal}.",
+            "8": "{user} is not {article} {animal}.",
+            "14": "{user} /might/ be {article} {animal}.",
+            "19": "{user} is DEFINITELY {article} {animal}.",
+            "20": "{user} is {article} CRITICAL {animal}!"
```
Comment (StarlitGhost) [inline desertbot/modules/automatic/Animals.py]: I think this article should always be 'a' since it's before CRITICAL

**Review by StarlitGhost — CHANGES_REQUESTED** [review]
I like it, both of the new modules look good. Couple of minor changes as already noted.

So much has changed in Reponses.py that it's hard to read the code through the diff, but I think those phrase responses are always going to be a bit janky given the weirdness of the matching.

**Comment by HubbeKing** [comment]
There we go. All cleaned up and hopefully easier to read now. Only thing I can think of that is removing the ResponseObject class and just using a dict instead, but that feels like a lot of work for very little benefit.

(As fun as having self.responses just be a dict of responseName to the stuff in self.bot.storage['responses'] would be, I think the ResponseObject class adds a lot of useful helper methods.)

**Pull request merged**